<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmServerManager</class>
 <widget class="QMainWindow" name="frmServerManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server Manager</string>
  </property>
  <widget class="QWidget" name="cwManager">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labTop">
      <property name="text">
       <string>Server Manager</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlManager">
      <item>
       <layout class="QVBoxLayout" name="vlManagerLeft">
        <item>
         <widget class="QScrollArea" name="saLeft">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="sawcLeft">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>256</width>
             <height>307</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="vlSawcLeft">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="lwServer">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlManagerRight">
        <item>
         <widget class="QScrollArea" name="saRight">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="sawcRight">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>143</width>
             <height>443</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="vlSawcRight">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <spacer name="vsSMU">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cmdAttach">
              <property name="text">
               <string>Attach</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdStart">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdStop">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdConfig">
              <property name="text">
               <string>Config</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdUpdate">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdAdmin">
              <property name="text">
               <string>Admin</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdCAttach">
              <property name="text">
               <string>Choose Attach</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdCStart">
              <property name="text">
               <string>Choose Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdCStop">
              <property name="text">
               <string>Choose Stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdCConfig">
              <property name="text">
               <string>Choose Config</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdCUpdate">
              <property name="text">
               <string>Choose Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdCIcon">
              <property name="text">
               <string>Choose Icon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdNewServer">
              <property name="text">
               <string>New Server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdDeleteServer">
              <property name="text">
               <string>Delete Server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdDAdmin">
              <property name="text">
               <string>Disable Admin</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="vsSMD">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
